extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -570,16 +568,15 @@ function axWFactorySaveVariable() {
 				 * catch parse errors
 				 */
 				ob_start();
-				if( eval( $tEval ) === FALSE ) {
+				if ( eval( $tEval ) === FALSE ) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this use of the "eval" function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1523)